src/backend/distributed/planner/multi_router_planner.c

```diff
@@ -417,10 +410,10 @@ RouterModifyTaskForShardInterval(Query *originalQuery, ShardInterval *shardInter
 			continue;
 		}
 
-		hashedOpExpressions = ShardIntervalOpExpressions(shardInterval, rteIndex);
-
-		hashedRestrictInfo = make_simple_restrictinfo((Expr *) hashedOpExpressions);
-		originalBaserestrictInfo = lappend(originalBaserestrictInfo, hashedRestrictInfo);
+		shardOpExpressions = ShardIntervalOpExpressions(shardInterval, rteIndex);
+		shardRestrictionList = make_simple_restrictinfo((Expr *) shardOpExpressions);
+		originalBaserestrictInfo = lappend(originalBaserestrictInfo,
```

Comment:
If you're renaming variables, could you also rename `originalBaserestrictInfo` to `originalBaseRestrictInfo` :)